datafusion/core/src/datasource/physical_plan/parquet.rs

```diff
@@ -178,7 +185,7 @@ mod tests {
                 source = source.with_bloom_filter_on_read(false);
             }
 
-            source.with_schema(Arc::clone(&file_schema))
+            source.with_schema(Arc::clone(&table_schema))
```

Comment:
👍 , I didn't notice here before

```rust
pub struct ParquetSource {
    /// Options for reading Parquet files
    pub(crate) table_parquet_options: TableParquetOptions,
    /// Optional metrics
    pub(crate) metrics: ExecutionPlanMetricsSet,
    /// The schema of the file.
    /// In particular, this is the schema of the table without partition columns,
    /// *not* the physical schema of the file.
    pub(crate) file_schema: Option<SchemaRef>,
```
Maybe we can change the `file_schema` name to a clearer name later.